modules/metrics/src/main/java/org/motechproject/metrics/event/MotechEventHandler.java

```diff
@@ -0,0 +1,67 @@
+package org.motechproject.metrics.event;
+
+import org.motechproject.event.MotechEvent;
+import org.motechproject.event.listener.annotations.MotechListener;
+import org.motechproject.metrics.api.Counter;
+import org.motechproject.metrics.service.MetricRegistryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * Handles the modification of metrics through the event bus or Tasks interface.
+ */
+@Service
+public class MotechEventHandler {
```

Comment:
Rename this class to indicate this is a handler for the metrics module, for example MetricsEventHandler